packages/manager/modules/pci/src/projects/project/storages/object-storage/users/add/user-add/user-add.controller.js

```diff
@@ -66,13 +66,28 @@ export default class PciUsersAddController {
         });
     }
     // action === 'createNewUser'
-    return this.createUser(user.description)
+    return this.createUser(this.newUserDescription)
       .then((data) => this.generateUserS3Credential(data))
       .finally(() => {
         this.isLoading = false;
       });
   }
 
+  isUserLinkedMode() {
+    return this.model === this.addExistingUser;
+  }
+
+  isUserCreationMode() {
+    return this.model === this.createNewUser;
+  }
+
+  isUserCanBeCreate() {
```

Comment:
```suggestion
  canUserBeCreated() {
```
using is + can in the same sentence is not correct